gke-scripts/00-common-env.sh

```diff
@@ -0,0 +1,79 @@
+#! /bin/sh
```

Comment:
Here and elsewhere:

```suggestion
#!/bin/sh
```

https://en.wikipedia.org/wiki/Shebang_(Unix)#Syntax

Reply:
@sanjaypujare there's still a bunch of places left with this extra space.